clients/include/solver/testing_getrf.hpp

```diff
@@ -95,12 +95,12 @@ void testing_getrf(const Arguments& arg)
     }
 
     // Naming: dK is in GPU (device) memory. hK is in CPU (host) memory
-    host_matrix<T>   hA(M, N, lda);
-    host_matrix<T>   hA1(M, N, lda);
-    host_matrix<int> hIpiv(1, Ipiv_size, 1);
-    host_matrix<int> hIpiv1(1, Ipiv_size, 1);
-    host_vector<int> hInfo(1);
-    host_vector<int> hInfo1(1);
+    host_matrix<T>       hA(M, N, lda);
+    host_matrix<T>       hA1(M, N, lda);
+    host_matrix<int>     hIpiv(1, Ipiv_size, 1);
```

Comment:
int64_t here? or Is it only for LAPACK ?

Reply:
Yeah the 32-bit is needed right now for rocSOLVER interface, 64-bit for reference-code interface